metaci/release/models.py

```diff
@@ -129,7 +135,14 @@ def _push_time(self, start):
 
 
 class Release(StatusModel):
-    STATUS = Choices("draft", "published", "hidden")
+    STATUS = Choices(
+        ("draft", _("Draft")),
+        ("failed", _("Failed")),
+        ("completed", _("Completed")),
+        ("inprogress", _("In Progress")),
+        ("waiting", _("Waiting")),
+        ("blocked", _("Blocked")),
+    )
```

Comment:
@davidmreed Are these the right choices? What needs to happen to existing values of published and hidden?

Reply:
Good callout, we're going to add a migration to map both removed statues to `completed` - the published and hidden distinction will be in MetaDeploy only.

Reply:
fwiw, there aren't actually any hidden releases in our production instance of MetaCI

Reply:
Yep, always possible some third-party operator has some though.

Reply:
I don't think we ever made "hidden" actually do anything though?

Reply:
Oh, huh. Django didn't notice this change and generate a migration at all, and we missed the `StatusModel` pattern when we implemented `ReleaseCohort` (which copies the use of `STATUS` but not the inheritance from `StatusModel`).

Given that it's a `CharField` under the hood, I guess it doesn't actually need a migration for the change to the `Choices`, just for mapping the old values.